dashboard/app/models/ability.rb

```diff
@@ -166,6 +166,17 @@ def initialize(user)
       end
     end
 
+    if user.permission?(UserPermission::LEVELBUILDER)
+      can :manage, [
+        FrequentUnsuccessfulLevelSource,
+        Level,
+        LevelSourceHint,
```

Comment:
Removed. Also FrequesntUnsuccessfulLevelSource. Though I don't see why we shouldn't give LBs those abilities.

Also added LevelSourceLevel, though again I'm unclear if this is necessary.